### ISSUE TITLE
Handle focusing in and out of table using useAccessibility hook

### ISSUE THREAD

**Issue by haxxmaxx** [qlik-oss/sn-table#10]
Where do we get focus initially when tabbing to the table? Do we keep tabIndex on a cell?
This might need updates in nebula, since I don't think there is a way to pass a callback to be called on an enter event on a sheet object (maybe not relevant for phase 1)

**Comment by haxxmaxx** [comment]
Given the incoming nebula hook `useAccessibility` we can detect when a chart is being focused and enter/space is pressed to focus in the table. Likewise if focus is stolen, we need to remove tabindex etc. If the user navigates out of the table (esc/tab to begin with) we just blur and reset tabindex. TBD if nebula should catch and keep track of this

**Comment by haxxmaxx** [comment]
List of requirements:
- when focusing the entire chart (grid cell), pressing enter should set focus on the top left cell
  - if you have already been inside the table, the focus should be set to the cell that was focused last
- clicking a cell should focus that cell and add a tabstop there
- when navigating to a new page, or the number of row changes (e.g confirming selections), the tabstop should be set to the top left cell again
- when pressing shift + tab from a cell, the focus should end up on the entire chart again. The tabstop should be removed so you can't tab inside the table again
- pressing esc should do the same thing as shift + tab
- changing the data from outside the table (e.g. changing selections from somewhere else) should change so that when you go back into the table, the top left cell should get focus, but not add a tabstop!

Basically whenever you exit the table using the keyboard, there should be no tabstop inside the table and it shouldn't be added before you enter it again.
Note that the footer will still have tabstops, that is a different issue.
